pil_analyzer/src/condenser.rs

```diff
@@ -103,6 +103,7 @@ impl<T: FieldElement> Condenser<T> {
         let (poly, _) = self
             .symbols
             .get(&reference.name)
+            .or_else(|| self.symbols.get(&format!("Global.{}", reference.name)))
```

Comment:
Why is `Global.` hardcoded here?

Reply:
Good point! I think this can be removed.